README.md

```diff
@@ -1,38 +1,9 @@
-# create-svelte
+# Ultimate Tutorial Series for SvelteKit
 
-Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/master/packages/create-svelte).
+Welcome to the Ultimate Tutorial Series for SvelteKit - you are in the right place to learn all about building a full-blown, multilingual website using Storyblok & SvelteKit. 
 
-## Creating a project
+Please follow the step by step instructions available in our documentation. You can find all the different parts on this overview page and take it from here. 
```

Comment:
Let's add the link to the tutorial here: https://www.storyblok.com/tp/the-storyblok-sveltekit-ultimate-tutorial

Reply:
updated, i update also the update-part-3  Pull Request :)